doc/directives.md

```diff
@@ -1,8 +1,8 @@
 # Directives
 
 `otk` has various directives that can be used in an omnifest. Generally these
-directives can appear anywhere in the tree and they are replaced with other
-trees or values.
+directives can appear anywhere in the tree unless otherwise specified and they
```

Comment:
```suggestion
directives can appear anywhere in the tree unless otherwise specified (see below) and they
```
those exceptions would be "specified" in this documentation, right?